recipes/gnu-time/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "gnu-time" %}
```

Comment:
Continuing the discussion [here]( https://github.com/conda-forge/staged-recipes/pull/3139#discussion-diff-122512038R46 ).